graph_adventure/adv.py

```diff
@@ -65,16 +65,35 @@ def oppositeCardinal(direction):
 #set key to starting room id with value equal to exits ex. 0: {n, e, s, w}
 visited[player.currentRoom.id] = player.currentRoom.getExits()
 print(visited)
-
 #have checked list for bidirectional travel back to the start of the map
 checked = []
-
 #track the steps to completion in order to pass tests/ set to traversalPath var
 count = []
-
+#while loop as long as visited is less than 500
+while len(list(visited)) < 499:
+    #if our current room has not been flagged, put it in visited 
```

Comment:
the way you went about this problem Austin looked like it was cake for you, great way of using your methods and also explaining each and every single functionality within the syntax!.